tests/test_mediastoredata/test_server.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import unicode_literals
+
+import sure  # noqa
+
+import moto.server as server
+from moto import mock_mediastoredata
+
+"""
+Test the different server responses
+"""
+
+
+@mock_mediastoredata
+def test_mediastore_lists_containers():
+    backend = server.create_backend_app("mediastoreback")
+    test_client = backend.test_client()
+
+    # res = test_client.get(
```

Comment:
Haven't tested it, but I think we don't need the headers in this case. `test_client.get("/")` should call `list_items()`, so the accompanying assertion would be that `result.should.contain('"Items": []')`